bearer/bearer.go

```diff
@@ -253,7 +253,7 @@ func (b Token) AssertUser(id user.ID) bool {
 // Note that any Token mutation is likely to break the signature, so it is
 // expected to be calculated as a final stage of Token formation.
 //
-// See also VerifySignature, Issuer.
+// See also VerifySignature, Issuer, [Token.SignedData].
```

Comment:
Doc links should be fixed at the same time.

Reply:
updated